src/observable.js

```diff
@@ -58,6 +60,7 @@ export class Observable {
     if (index > -1) {
       this.handlers_.splice(index, 1);
     }
+    dev().error('Can\'t remove nonexistent handler %s.', handler);
```

Comment:
We should keep this error at the story level like you did before, otherwise it might start throwing everywhere in the AMP codebase

Reply:
I'll remove it, but we can't check at the story level without making bigger changes. 

What we currently have available at the story level (in the store) is `this.listeners_`. Which is a map of type <`StateProperty`, `Observable`>. The `Observable` is the one containing the array of `listeners` that each `StateProperty` contains. 

To check if a given `StatePropertyZ` has certain `listenerX`, we would need to keep track of all listeners added to `StatePropertyZ` at the story level (in the store). 

What I was checking before and what I'm including again is whether `StateProperty` has any listeners at all before calling `Observable#remove()`.